projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/group-skipper/group-skipper.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  GroupSkipperPageConfig,
+  GroupSkipperConfig,
+} from './group-skipper.config';
+
+export function testGroupSkipperConfig(config: GroupSkipperConfig) {
```

Comment:
rename to `testGroupSkipperFromConfig` - test..config implies that we are testing the config itself

Reply:
on the other hand, this could be `testAllGroupSkipperPagesFromConfig` (too long, but something similar), and `testGroupSkipperPageConfig` could be then renamed to `testGroupSkipperFromConfig`